### PR TITLE
Push notification when user sent a new message in a conversation

### PR DESCRIPTION
**Pull request by enzofab91**
**### Feature:**
As a user I should be able to receive notifications when someone sends me a message so I can be aware of it.

### **Trello reference:**
https://trello.com/c/S99cBcOx

### **Description:**
This PR adds a push notification when a message is sent in a conversation.

### CODE REVIEW

File: app/models/conversation.rb [hunk 1]
```diff
@@ -26,6 +26,10 @@ class Conversation < ApplicationRecord
 
   validate :users_uniqueness, on: :create
 
+  def other_user(user)
```
Comment (mrodriguez90) [inline app/models/conversation.rb:29]: This is a little bit ambiguous, maybe recipient_user, or doesn't make any sense here?

File: app/models/conversation.rb [hunk 2]
```diff
@@ -26,6 +26,10 @@ class Conversation < ApplicationRecord
...
+    users.where.not(id: user.id).first
```
Comment (mrodriguez90) [inline app/models/conversation.rb:30]:
```suggestion
    users.where.not(id: user.id)&.first
```

**Review by mrodriguez90 — APPROVED** [review]

File: app/models/message.rb [hunk 3]
```diff
@@ -26,6 +26,7 @@ class Message < ApplicationRecord
   validates :body, presence: true
 
   after_create_commit :broadcast_message
+  after_create_commit :notify_user
```
Comment (sebastiancaraballo) [inline app/models/message.rb:29]: When possible, avoid adding callbacks to models. Instead, try extracting the logic into the service or class from where the message is created. Just for your information.

**Review by sebastiancaraballo — APPROVED** [review]

File: app/models/conversation.rb (hunk 2 above)
Comment (enzofab91) [reply app/models/conversation.rb:30]: Whats & for?

Comment (sebastiancaraballo) [reply app/models/conversation.rb:30]: It's a way to avoid a null error. Take a look at [this](https://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/).

**Pull request merged**